src/web/components/Onwards/Carousel/CarouselCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Card } from '@frontend/web/components/Card/Card';
+import { LI } from '@frontend/web/components/Card/components/LI';
+
+type CarouselCardProps = {
+	trail: TrailType;
+};
+
+export const CarouselCard: React.FC<CarouselCardProps> = ({
+	trail,
+}: CarouselCardProps) => {
+	const {
+		pillar,
+		design,
+		url: linkTo,
+		image: imageUrl,
+		headline: headlineText,
+		webPublicationDate,
+		kickerText,
+	} = trail;
+	return (
+		<LI stretch={true} percentage="100%" showDivider={true} padSides={true}>
+			<Card
+				linkTo={linkTo}
+				pillar={pillar}
+				design={design}
+				headlineText={headlineText}
+				webPublicationDate={webPublicationDate}
+				kickerText={kickerText || ''}
+				imageUrl={imageUrl || ''}
+				showClock={true}
+				alwaysVertical={true}
+				minWidthInPixels={258}
+			/>
+		</LI>
```

Comment:
A take it leave it thought but you could just inline this code directly inside the `ul`. It would make that file a big bigger but it would reduce the need to swap files. But it's true `Carousel` is getting big so I'll leave it up to you.

One thing though, and this applies in both cases. On the first `LI` if you set `showDivider` to false then you don't get the leading vertical divider.

You could achieve this with some sort of `isFirst` prop as it stands but if it were to inline the map then you could just `showDivider={index !== 0}`?